cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -210,13 +211,55 @@
                 {{ notification('warning', true,
                     'Sorry, there were no results based on your filter selections.',
                     'Please reset the filter or change your selections and try again.') }}
+            {% elif posts.paginator.count == 1 %}
```

Comment:
Oh, there's another place we do this above on [line 208](https://github.com/cfpb/cfgov-refresh/pull/2587/files#diff-6968a1d9f0cd02ef3900aad4ecc1b09aL208).